drivers/gpio/gpio_mcux_rgpio.c

```diff
@@ -80,6 +80,51 @@ static int mcux_rgpio_configure(const struct device *dev,
 			((size_t)config->pin_muxes[cfg_idx].config_register);
 	uint32_t reg = *gpio_cfg_reg;
 
+#if defined(CONFIG_SOC_SERIES_IMXRT118X)
+	if (config->pin_muxes[pin].pue_mux) {
+		/* PUE type register layout (GPIO_AD pins) */
+		if ((flags & GPIO_SINGLE_ENDED) != 0) {
+			/* Set ODE bit */
+			reg |= IOMUXC_SW_PAD_CTL_PAD_ODE_MASK;
+		} else {
+			reg &= ~IOMUXC_SW_PAD_CTL_PAD_ODE_MASK;
+		}
+
+		if (((flags & GPIO_PULL_UP) != 0) || ((flags & GPIO_PULL_DOWN) != 0)) {
+			reg |= IOMUXC_SW_PAD_CTL_PAD_PUE_MASK;
+			if (((flags & GPIO_PULL_UP) != 0)) {
+				reg |= IOMUXC_SW_PAD_CTL_PAD_PUS_MASK;
+			} else {
+				reg &= ~IOMUXC_SW_PAD_CTL_PAD_PUS_MASK;
+			}
+		} else {
+			/* Set pin to highz */
+			reg &= ~IOMUXC_SW_PAD_CTL_PAD_PUE_MASK;
+		}
+	} else {
+		/* PDRV/SNVS/LPSR type register layout */
+		if (((flags & GPIO_PULL_UP) != 0) || ((flags & GPIO_PULL_DOWN) != 0)) {
+			reg &= ~IOMUXC_SW_PAD_CTL_PAD_PULL_MASK;
+			if (((flags & GPIO_PULL_UP) != 0)) {
+				reg |= IOMUXC_SW_PAD_CTL_PAD_PULL(0x1U);
+			} else {
+				reg |= IOMUXC_SW_PAD_CTL_PAD_PULL(0x2U);
+			}
+		} else {
+			/* Set pin to no pull */
+			reg |= IOMUXC_SW_PAD_CTL_PAD_PUS_MASK;
```

Comment:
this is wrong, see https://github.com/zephyrproject-rtos/zephyr/pull/75328

Reply:
Fixed, thank you for reminding.